packages/generators/utils/entry.ts

```diff
@@ -18,19 +18,18 @@ interface CustomGenerator extends Generator {
 
 export default function entry(
 	self: CustomGenerator,
-	answer: {
-		entryType: boolean;
-	}
+	multiEntries: boolean,
 ): Promise<void | {}> {
 	let entryIdentifiers: string[];
 	let result: Promise<void | {}>;
-	if (answer.entryType) {
+	if (multiEntries) {
 		result = self
 			.prompt([
 				InputValidate(
 					"multipleEntries",
-					"Type the names you want for your modules (entry files), separated by comma [example: app,vendor]",
-					validate
+					"How do you want to name your bundles? (separated by comma)",
```

Comment:
Example here is probably a good idea as this is the confusing part of webpack

Reply:
I've removed the example from the prompt in favor of default answers. It now looks like this:

![bundles](https://user-images.githubusercontent.com/11232797/58471480-419bce80-8144-11e9-86c7-210f3df6228c.png)